R/reliabilityAnalysisFrequentist.R

```diff
@@ -0,0 +1,606 @@
+                                  "mcDonaldScalef", "alphaScalef", "guttman2Scalef", "guttman6Scalef", "glbScalef", 
+                                  "averageInterItemCor", "meanScale", "sdScale",
+                                  "mcDonaldItemf",  "alphaItemf",  "guttman2Itemf", "guttman6Itemf", "glbItemf",
+                                  "reverseScaledItems", "meanItem", "sdItem", "itemRestCor", "missingValuesf", 
+                                  "omegaEst", "alphaMethod", "setSeed", "seedValue"))
+  itemTableF$addColumnInfo(name = "variable", title = "Item", type = "string")
+  
+  idxSelectedF <- which(itemDroppedSelectedF)
+  coefficients <- derivedOptions[["namesEstimators"]][["coefficients"]]
+  for (i in idxSelectedF) {
+    if (estimators[i] %in% coefficients) {
+      itemTableF$addColumnInfo(name = paste0("pointEst", i), title = estimators[i], type = "number", 
+                               overtitle = overTitle)
+    } else {
+      itemTableF$addColumnInfo(name = paste0("pointEst", i), title = estimators[i], type = "number")
+    }
+  }
+  
+  relyFit <- model[["relyFit"]]
+  if (!is.null(relyFit)) {
+    tb <- data.frame(variable = model[["itemsDropped"]])
+    for (i in idxSelectedF) {
+      newtb <- cbind(pointEst = relyFit$freq$ifitem[[i]])
+      colnames(newtb) <- paste0(colnames(newtb), i)
+      tb <- cbind(tb, newtb)
+      
+    }
+    itemTableF$setData(tb)
+    
+  } else if (length(model[["itemsDropped"]]) > 0) {
+    itemTableF[["variables"]] <- model[["itemsDropped"]]
+  }
+  
+  jaspResults[["itemTableF"]] <- itemTableF
+  return()
+}
+
+# once the package is updated check this again and apply:
+.freqentistReliabilitySingleFactorFitTable <- function(jaspResults, model, options) {
+
+  if (!options[["fitMeasures"]] || !options[["mcDonaldScalef"]])
+    return()
+  if (!is.null(jaspResults[["fitTable"]]) || !options[["fitMeasures"]]) {
+    print("jaspResults[['fitTable']] from state or not wanted")
+    return()
+  }
+
+  fitTable <- createJaspTable(sprintf("Fit Measures of Single Factor Model Fit"))
+  fitTable$dependOn(options = c("variables", "mcDonaldScalef", "reverseScaledItems", "fitMeasures", "missingValuesf", 
+                                "omegaEst", "setSeed", "seedValue"))
+  fitTable$addColumnInfo(name = "measure", title = "Fit Measure",   type = "string")
+  fitTable$addColumnInfo(name = "value",     title = "Value", type = "number")
+
+  relyFit <- model[["relyFit"]]
+  derivedOptions <- model[["derivedOptions"]]
+  opts <- names(relyFit$freq$omega_fit)
+
+  if (!is.null(relyFit)) {
+    if (is.null(opts)) {
+      allData <- data.frame(
+        measure = NA_real_,
+        value = NA_real_
+      )
+      if (!is.null(relyFit[["freq"]][["omega.error"]])) {
+          fitTable$addFootnote("Fit measures cannot be displayed because the omega estimation method switched 
+                                to PFA as the CFA did not find a solution.")
```

Comment:
```suggestion
          fitTable$addFootnote(gettext("Fit measures cannot be displayed because the omega estimation method switched 
                                to PFA as the CFA did not find a solution."))
```